io.github.lpub3d.LPub3D.json

```diff
@@ -0,0 +1,183 @@
+{
+  "id": "io.github.lpub3d.LPub3D",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15",
+  "sdk": "org.kde.Sdk",
+  "command": "lpub3d24",
+  "rename-icon": "lpub3d",
+  "rename-desktop-file": "lpub3d.desktop",
+  "rename-appdata-file": "io.github.lpub3d.appdata.xml",
+  "build-options": {
+    "env": {
+      "LP3D_3RD_PARTY_DIR": "$(realpath ./lpub3d_linux_3rdparty)",
+      "LP3D_DIST_DIR_PATH": "$(realpath ./lpub3d_linux_3rdparty)",
+      "LDRAWDIR": "$(realpath ./lpub3d_linux_3rdparty)/ldraw",
+      "PLATFORM_CODE": "osl",
+      "TARGET_CPU": "$(uname -m)",
+      "PLATFORM_VER": "$(. /etc/os-release 2>/dev/null; [ -n \"$VERSION_ID\" ] && echo $VERSION_ID || true)"
```

Comment:
This probably won't work as intended either.
```
$ echo $PLATFORM_VER
$(. /etc/os-release 2>/dev/null; [ -n "$VERSION_ID" ] && echo $VERSION_ID || true)
```

Reply:
I will have to find another way to populate this var. I’ll pass it directly to `qmake` in `config-ops`

Cheers,